src/main.py

```
@@ -1570,6 +1614,13 @@ async def lifespan(app: FastAPI):
     # 6. Global clients cleanup
     await clients.cleanup()
 
+    # 6b. SQL engine cleanup
+    try:
+        from db.engine import dispose_engine
+        await dispose_engine()
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)